src/meta/src/controller/fragment.rs

```diff
@@ -1096,7 +1096,7 @@ impl CatalogController {
         fragment_id: FragmentId,
     ) -> MetaResult<Vec<(ActorId, Vec<ActorId>)>> {
         let inner = self.inner.read().await;
-        let actors: Vec<(ActorId, Vec<ActorId>)> = Actor::find()
+        let actors: Vec<(ActorId, ActorUpstreamActors)> = Actor::find()
```

Comment:
How did you find this is wrong? 🤡

Reply:
I was running some tests locally and the meta failed to restart when `load_backfill_fragment_ids`. 🥵

Reply:
😄 no worries, these part of codes are not in the release 1.8.

Reply:
Yes, this can be triggered whenever any fragment is created (why?), like `CREATE TABLE t()`, and then restart risingwave.